functions/Set-DbaDbFileGrowth.ps1

```diff
@@ -29,6 +29,9 @@ function Set-DbaDbFileGrowth {
     .PARAMETER Growth
         The growth value. 64 by default.
 
+    .PARAMETER FileType
+        Apply changes to only DATA, LOG or ALL data files
```

Comment:
Maybe remove the "data"? "Apply changes to only DATA, LOG or ALL files"

Reply:
Made the data change.  I do not have my configuration setup to test the TESTS, so not sure what to add.